protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/ThermocyclerTools/ProfileStepsSummary.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
```

Comment:
https://github.com/Opentrons/opentrons/pull/16353

```suggestion
import { useState } from 'react'
```